New checkout flow (#68)
Rythm-bot/docs: docs/Premium.md

:::
## How to get Rythm Premium perks?
---
1. Go to https://rythm.fm/premium#plans and choose a preferred plan. The page will prompt you to login if you haven't previously.

> 2. Choose a payment method: Credit/debit card or Paypal.

Reviewer: ```suggestion
2. Choose a payment method: Credit/debit card or PayPal.
```

<Tabs
  defaultValue="credit"
  values={[
    {label: 'Credit/debit card', value: 'credit'},

>     {label: 'Paypal', value: 'paypal'},

Reviewer: ```suggestion
    {label: 'PayPal', value: 'paypal'},
```

  ]}>
  <TabItem value="credit">
    <ul>
      <li>Fill in your credit card information. Click <code>Next</code> to proceed.</li><br/>
      <img src="/docs/img/docs/premium/credit-payment-details.png" alt="credit-payment-details"/>

>       <li>Fill in the required field with your private information. Click <code>Next</code> to proceed.</li><br/>

Reviewer: ```suggestion
      <li>Fill in the required fields with your billing information. Click <code>Next</code> to proceed.</li><br/>
```

      <img src="/docs/img/docs/premium/credit-billing-information.png" alt="credit-billing-information"/>

>       <li>Confirm your payment method, billing information, subscription plan, coupon code (if you have one). Press <code>Checkout</code> and we are good to go.</li><br/>

Reviewer: ```suggestion
      <li>Confirm your payment method, billing information, subscription plan, and coupon code (if you have one). Press <code>Checkout</code> to make your payment.</li><br/>
```

      <img src="/docs/img/docs/premium/credit-review.png" alt="credit-review"/>

>     </ul>

Reviewer: ```suggestion
      <li>Complete the secure payment prompt to authorize your card if required, and then your payment will be completed and you'll be sent to the dashboard to start using Rythm Premium.</li>
    </ul>
```
Can reword if you want, but most folks will get a 3ds prompt at this point

  </TabItem>

  <TabItem value="paypal">
    <ul>

>       <li>Log into your Paypal account and choose your preferred Credit for your payment. Click <code>Next</code> to proceed.</li><br/>

Reviewer: ```suggestion
      <li>Log into your PayPal account and choose your preferred funding method for your payment. Click <code>Next</code> to proceed.</li><br/>
```

      <img src="/docs/img/docs/premium/paypal-payment-details.png" alt="paypal-payment-details"/>

>       <li>The page will automatically fill in the required information. You will just need to confirm your payment method, billing information, subscription plan, coupon code (if you have one). Press <code>Checkout</code> and we are good to go.</li><br/>

Reviewer: ```suggestion
      <li>Your billing information will be automatically filled out, so you won't have to complete that page. You will just need to confirm your payment method, billing information, subscription plan, and coupon code (if you have one). Press <code>Checkout</code> to complete your payment and you'll be sent to the dashboard to begin using Rythm Premium.</li><br/>
```

      <img src="/docs/img/docs/premium/paypal-review.png" alt="paypal-review"/>
    </ul>
  </TabItem>
</Tabs>

:::tip Note

> An email with invoice will be sent to you if the purchase is successful!

Reviewer: ```suggestion
An email with an invoice will be sent to you if the purchase is successful!
```
